.travis.yml

```diff
@@ -51,7 +51,7 @@ script:
    #run template app generation
    - travis_wait 30 set -o pipefail && node external/shared/node/test_force.js --os=android --test=native
 
-   - travis_wait 30 ./gradlew createDebugCoverageReport
+   - travis_wait 50 ./gradlew createDebugCoverageReport
```

Comment:
since we added some new tests, especially the load test, the test need longer time to complete.